### ISSUE TITLE
Need Browser Support in README

### ISSUE THREAD

**Issue by cwtuan** [Matt-Esch/virtual-dom#336]
Does it support IE8?

**Comment by aphillipo** [comment]
Yeah, I need to know this too.  I guess I need to support IE8 still...

**Comment by Raynos** [comment]
Yes. Ie8 is supported. Browser support can be seen in build status.

However it looks like saucelabs badge has since broken.

Virtual Dom supports ie6 and up. If something does not work, please open an issue reporting the bug.

**Comment by aphillipo** [comment]
That's great to know!  Thanks.

**Comment by keenwon** [comment]
@Raynos 
IE6+, awsome
but in package.json the testling.browser is ie8+
